### PR TITLE
Pa 4 - shelter public profile

### PR DESCRIPTION
**Pull request by avb49**
Incorporated logic in ShelterPublicProfileActivity to load attributes from a given shelter into the profile. Textviews and editTextviews are set to non-editable.
Changes in layout in the relevant xml file.

### CODE REVIEW

File: app/src/main/java/uk/ac/bath/petmatch/Database/ShelterDao.java [hunk 1]
```diff
@@ -11,6 +11,15 @@ public ShelterDao(Dao<Shelter, String> dao) {
         super(dao);
     }
 
+    public User findById(String shelterId) {
+        List<User> results = this.queryForEq("id", shelterId);
```
Comment (novotnej) [inline app/src/main/java/uk/ac/bath/petmatch/Database/ShelterDao.java]: This is a list with <Shelter> items

**Review by novotnej — CHANGES_REQUESTED** [review]
The address end e-mail fields still look as editable fields
Remove the android picture and move up the information
Use the real charity number instead of a dummy
Add Phone to the model and database
Make e-mail clickable, to open an e-mail client and sent e-mail to that address (MailTo)

Move the details up a bit and put a map under it with a pin in the gps coordinates

**Comment by avb49** [comment]
Changes have been done. 
Remember to log in as shelter@petmatch.com to see the shelter profile details being retrieved and displayed. Otherwise, some default field strings are displayed on the screen and the map has no pin.

**Review by novotnej — CHANGES_REQUESTED** [review]
This looks really good, could you please just use the logged in version? 

Also please merge master into it, there's some conflicts

Also just in case you wanted to and definitely a nice-to-have, but would it be possible to have a "navigate to" button in the shelter? I think it might be a feature of the maps view, or of the marker, but not sure

**Comment by avb49** [comment]
When you say merge master, what exactly do you mean?
Should I just do 'git merge master' on the PA-4 branch?

**Comment by novotnej** [comment]
yep exactly that, if you want to merge anything into master (or any other branch for that matter) first you merge master into it (into PA-4) and resolve the conflicts, then you have to make sure everything is still working and then it can be all merged into master

**Comment by avb49** [comment]
alright cool. Will do

**Comment by avb49** [comment]
Should be good now.

**Comment by novotnej** [comment]
Did couple of changes, namely replaced the edit views with textviews, merging it now

**Review by novotnej — APPROVED** [review]

**Pull request merged**